xbmc/platform/win32/input/IRServerSuite.cpp

```diff
@@ -76,14 +76,14 @@ void CIRServerSuite::Process()
 {
   m_profileId = CServiceBroker::GetProfileManager().GetCurrentProfileId();
   m_irTranslator.Load(IRSS_MAP_FILENAME);
+  unsigned int nRetryCount(6);
 
   // try to connect 60 times @ a 5 second interval (5 minutes)
   // multiple tries because irss service might be up and running a little later then xbmc on boot.
-  while (!m_bStop)
+  while (!m_bStop && nRetryCount--)
```

Comment:
this makes imposible to re-connect to IRSS after disconnection in case if it connected on the last try, please reset try count after success connect.

Reply:
Can we make IRSS optional in (Input) settings pls?